.github/workflows/crowdin-i18n.yml

```diff
@@ -0,0 +1,92 @@
+name: Crowdin Sync
+on:
+  workflow_dispatch:
+  # schedule:
+  # (TODO?) run this Action every 14 days
+  # - cron: '0 * */14 * *'
+  push:
+    branches:
+      - i18n-dev
+      # Treat the below branches as special case for working on workflows
+      - actions-**
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-18.04
+
+    strategy:
+      matrix:
+        node-version: [12.x]
+
+    steps:
+      - name: Checkout Source Files
+        uses: actions/checkout@v2
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: Cache node modules
+        uses: actions/cache@v2
+        env:
+          cache-name: cache-node-modules
+        with:
+          # npm cache files are stored in `~/.npm` on Linux/macOS
+          path: ~/.npm
+          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-build-${{ env.cache-name }}-
+            ${{ runner.os }}-build-
+            ${{ runner.os }}-
+
+      - name: Set Environment variables
+        run: cp sample.env .env
+
+      - name: Lint Source Files
+        run: |
+          npm ci
+          npm run ensure-env --if-present
+          npm run lint --if-present
```

Comment:
On second thoughts we can avoid running linting here. It's not under the scope of the workflow technically.